frontend/public/style/_overrides.scss

```
@@ -358,6 +358,7 @@ ul {
 ul.pf-c-alert-group,
 ul.pf-c-nav__list,
 ul.pf-c-simple-list__list,
-ul.pf-c-tree-view__list {
+ul.pf-c-tree-view__list,
+.pf-c-pagination ul.pf-c-options-menu__menu {
```

This can simply be
```suggestion
ul.pf-c-options-menu__menu {
```
no?

Also, please sort the list of selectors alphabetically rather than simply appending to the bottom.

---

👍 Updated to just be `ul.pf-c-options-menu__menu` and sorted. Obviously this will now affect all PatternFly options menus, not just pagination menus, but that seems like the what we would want.